ew/cmd/fish/fishutils.py

```diff
@@ -511,6 +515,27 @@ async def award_fish(fisher, cmd, user_data):
         fisher.stop()
 
         user_data.persist()
+
+        #GoonScape Stat
+        xp_map = {
+            "item": 21000,
+            "common": 11000,
+            "uncommon": 16000,
+            "rare":	26000,
+            "promo": 31000,
+        }
+
+        if fisher.inhabitant_id:
+            xp_yield = xp_map.get(xp_type/2, 16000/2)
+            await add_xp(cmd.message.author.id, cmd.message.guild.id, cmd.message.channel, goonscape_fish_stat, xp_yield)
+
+            xp_yield = xp_map.get(xp_type/2, 16000/2)
+            await add_xp(inhabitant_id, cmd.message.guild.id, cmd.message.channel, goonscape_fish_stat, xp_yield)
+        else: 
+            xp_yield = xp_map.get(xp_type, 16000)
+            xp_yield = xp_map.get(xp_type, 18000000)
```

Comment:
is this a typo? 18 mil experience is like twice the amount you need for level 99 and it overwrites the first assign right before it